test/cookbooks/osl-git-test/recipes/osl_gitlfs.rb

```diff
@@ -19,13 +19,25 @@
 # This recipe creates a git_credentials resource for testing in ChefSpec.
 # Attributes allow changing values for different test contexts.
 
+
+pseudoGroup = 'nobody'
+if platform_family?('debian')
+  pseudoGroup = 'nogroup' 
+
+  directory '/tmp/bar' do
+    owner 'nobody'
+    group 'nogroup'
+    mode '0755'
+  end
+end
+
```

Comment:
This whole block can be removed by what I did above. I'm confused why you're creating the directory however.

Reply:
The /tmp/bar directory was getting "dubious ownership" git errors when switching from group 'nobody' to 'nogroup' which cancelled the command execution. This is just a way I found around that error